### PR TITLE
Automated PR for W3C-19980720

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/W3C-19980720.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+         modifications, that you make:</p>
+      <list></list>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>The full text of this NOTICE in a location viewable to users of the redistributed or derivative work.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Any pre-existing intellectual property disclaimers, notices, or terms and conditions. If none
```
Comment (pmadick) [inline src/W3C-19980720.xml]: The body of this license starts at least at line 16 and probably at line 15.  Please check with J.

Comment (jlovejoy) [inline src/W3C-19980720.xml]: agreed.  I think we can simply remove the optional tag and move the body tag up (check that is correct change as to body tag)

File: src/W3C-19980720.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+         FOR ANY PARTICULAR PURPOSE OR THAT THE USE OF THE SOFTWARE OR DOCUMENTATION WILL NOT INFRINGE ANY
+         THIRD PARTY PATENTS, COPYRIGHTS, TRADEMARKS OR OTHER RIGHTS.</p>
+      <p>COPYRIGHT HOLDERS WILL NOT BE LIABLE FOR ANY DIRECT, INDIRECT, SPECIAL OR CONSEQUENTIAL DAMAGES ARISING
+         OUT OF ANY USE OF THE SOFTWARE OR DOCUMENTATION.</p>
+      <p>The name and trademarks of copyright holders may NOT be used in advertising or publicity pertaining to
+         the software without specific, written prior permission. Title to copyright in this software and any
+         associated documentation will at all times remain with copyright holders.</p>
+    </body>
+    <optional>
+      <p>____________________________________</p>
```
Comment (jlovejoy) [inline src/W3C-19980720.xml]: we did not have this marked as optional before - should we do so now or leave it as part of the license?

File: src/W3C-19980720.xml (hunk 1 above)
Comment (bradleeedmondson) [inline src/W3C-19980720.xml]: agree w/ @jlovejoy

File: src/W3C-19980720.xml (hunk 2 above)
Comment (bradleeedmondson) [inline src/W3C-19980720.xml]: Coming at this fresh, I would say this following should be optional. But I would also say that it's fine to leave it as-is for now and review whether to apply the &lt;optional&gt; tag more aggressively the in future.

Comment (jlovejoy) [inline src/W3C-19980720.xml]: yes, ok as optional, discussed on Aug 18 legal call

**Pull request merged**